html/views/overlay/index.js

```diff
@@ -191,7 +151,26 @@ function initialize() {
 			setTimeout(function() { $('body').removeClass('preload'); }, 1000);
 }
 
-
+function jammer_ov(k, v) {
```

Comment:
This is still duplicating, try to share as much code as you sanely can with the existing function.

I very much look forward to cleaning all of this up, but that's a bigger effort.

Reply:
I tryed now to use the orginal jammer function by calling it from the jammer_ov function and handing over a bool. That seems to work without breaking the original function.